compiler/one-cmds/tests/one-partition_001.test

```diff
@@ -33,7 +33,7 @@ outputfile="${testmodel}.conn.json"
 rm -f ${filename}.log
 rm -rf  ${testmodel}.000*
 rm -rf  ${testmodel}.conn.*
```

Comment:
Q) these are left `-rf`. are these folders ?

Reply:
No. I just wasn't sure it wasn't a folder:) Let me cover them, too.